src/assets/javascript/analytics/ga4/pageViewTracker.js

```diff
@@ -0,0 +1,54 @@
+/* global window document */
+
+window.DI = window.DI || {}
+window.DI.analyticsGa4 = window.DI.analyticsGa4 || {}
+window.DI.analyticsGa4.trackers = window.DI.analyticsGa4.trackers || {};
+
+(function (trackers) {
+
+  'use strict'
+
+  let PageViewTracker = {
+
+    init: function() {
+      const data = {
+        event: 'page_view',
+        page_view: {
+          language: this.getLanguage(),
+          location: this.getLocation(),
+          organisations: '<OT1056>',
+          primary_publishing_organisation: 'government digital service - digital identity',
+          status_code: this.getStatusCode(),
+          title: this.getTitle(),
+          referrer: this.getReferrer(),
+          taxonomy_level1: 'web cri',
+          taxonomy_level2: 'pre cri'
```

Comment:
This has been slightly modified, to change the taxonomy and fetch the language from the html attribute